main.go

```diff
@@ -24,6 +24,8 @@ var (
 	debug       = flag.Bool("debug", false, "sets log to debug level")
 	healthAddr  = flag.String("health_addr", ":8080", "configure http listener for reporting health")
 	selfVersion = flag.Bool("version", false, "prints the version information")
+	vaultAddr   = flag.String("vault_addr", "https://127.0.0.1:8200", "default address for connecting to Vault")
```

Comment:
Now that I look at this closely, `--health_addr` should really be `--health-addr` to be consistent with the rest of our tools. Please could you switch to dashes? And I'll make changes separately to fix `health_addr`.

Reply:
Done!